tests/auction/test_auction_house.py

```diff
@@ -728,6 +728,18 @@ def test_settle_auction_with_bid(token, deployer, auction_house_unpaused, alice)
     assert deployer_balance_after == deployer_balance_before + 100
 
 
+def test_settle_current_and_create_new_auction_with_bid_smart_contract_owner(token, smart_contract_owner, auction_house_sc_owner, alice):
```

Comment:
You can test that this reverts if you change `raw_call` back to `send` in `_settle_auction`